orchestration/dap_orchestration/solids.py

```diff
@@ -241,8 +242,59 @@ def write_outfiles(context: AbstractComputeExecutionContext, fan_in_results: lis
     NOTE: The fan_in_results param allows to introduce a fan-in dependency from upstream transformation
     solids, but is ignored by this solid.
     """
+    tsv_directory = context.solid_config["output_dir"]
     context.resources.outfiles_writer.run(
-        context.solid_config["output_dir"],
+        tsv_directory,
         context.resources.refresh_directory,
         fan_in_results
     )
+
+    return tuple((fan_in_results, tsv_directory))
+
+
+@solid(
+    required_resource_keys={"refresh_directory", "gcs"},
+    config_schema={
+        "upload_dir": String,
```

Comment:
⛏️ I'd call this `destination_gcs_path` as we're not really uploading to a dir but copying between GCS paths.